docs/components/tabs.md

```diff
@@ -0,0 +1,353 @@
+            bottom: TabBar(
+              indicatorColor: Colors.white,
+              tabs: [
+                Tab(text: 'DOGS', icon: Icon(Icons.favorite)),
+                Tab(text: 'CATS', icon: Icon(Icons.music_note)),
+                Tab(text: 'BIRDS', icon: Icon(Icons.search)),
+              ],
+            ),
+          ),
+          body: TabBarView(
+            children: [
+                Center(child: Text('DOGS')),
+                Center(child: Text('CATS')),
+                Center(child: Text('BIRDS')),
+            ],
+          ),
+        ),
+      ),
+    );
+```
+
+## Scrollable tabs
+
+Scrollable tabs are displayed without fixed widths. They are scrollable, such that some tabs will remain off-screen until scrolled.### Fixed tab example
+
+`Tabs`
+
+* [Class definition](https://api.flutter.dev/flutter/material/TabBar-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/tabs.dart)
+* [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/tabs/scrollable)
+* [Demo site](https://gallery.flutter.dev/#/demo/tabs)
+
+The following example shows a row of of scrollable tabs.
+
+![Scrollable tab example with 9 tabs.](assets/tabs/tabs_scrollable.png)
+
+```dart
+    final tabs = [
+      'Tab 1',
+      'Tab 2',
+      'Tab 3',
+      'Tab 4',
+      'Tab 5',
+      'Tab 6',
+      'Tab 7',
+      'Tab 8',
+      'Tab 9',
+    ];
+
+    return MaterialApp(
+      title: 'Flutter Demo',
+      home: DefaultTabController(
+        length: tabs.length,
+        child: Scaffold(
+          appBar: AppBar(
+            title: Text('Scrollable Tabs'),
+            automaticallyImplyLeading: false,
+            backgroundColor: Color(0xff5808e5),
+            bottom: TabBar(
+              indicatorColor: Colors.white,
+              isScrollable: true,
+              tabs: [
+                for (final tab in tabs) Tab(text: tab),
+              ],
+            ),
+          ),
+          body: TabBarView(
+            children: [
+              for (final tab in tabs)
+                Center(
+                  child: Text(tab),
+                ),
+            ],
+          ),
+        ),
+      ),
+    );
+```
+
+## Theming tabs
+
+`Tabs`
+
+* [Class definition](https://api.flutter.dev/flutter/material/TabBar-class.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/tabs.dart)
+* [Dartpad demo for Scrollable Theme](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/tabs/scrollable_theme)
+* [Dartpad demo for Fixed Theme](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/tabs/theme)
+
+The following example shows a row of of scrollable tabs.
```

Comment:
Mention that this uses the [Shrine](https://material.io/design/material-studies/shrine.html#when-to-adapt) color theme.

Reply:
Done